File: year2/semester2/mpp/c#/TripClientServer/TripCommon/Domain/Account.cs
using System;
using TripCommon.Domain;

namespace TripCommon.domain
{
    public class Account: Entity<int>
    {
        private String name; 
        private String password;

        public override string ToString()
        {
            return Id + " " + name + " " + password;
        }

        public Account()
        {
        }
        public Account(int id, string name, string password)
        {
            this.Id = id;
            this.Name = name;
            this.Password = password;
        }
        public Account(string name, string password)
        {
            this.Name = name;
            this.Password = password;
        }

        public string Name { get => name; set => name = value; }
        public string Password { get => password; set => password = value; }
    }
}

File: year2/semester2/mpp/java/TripClientServerSpringRemoting/TripCommon/src/main/java/domain/Account.java
package domain;

import java.io.Serializable;
import java.util.Objects;

public class Account extends Entity<Integer> implements Serializable {
    private String name;
    private String password;

    public Account(Integer id, String name, String password) {
        this.id = id;
        this.name = name;
        this.password = password;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        Account account = (Account) o;
        return name.equals(account.name) &&
                password.equals(account.password);
    }

    @Override
    public int hashCode() {
        return Objects.hash(name, password);
    }

    @Override
    public String toString() {
        return id + " " + name + " " + password;
    }

    public Account() {
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
}
